tests/xen/conftest.py

```diff
@@ -65,3 +66,11 @@ def host_with_dom0_tests(host_with_saved_yum_state):
     host = host_with_saved_yum_state
     host.yum_install(['xen-dom0-tests'])
     yield host
+
+@pytest.fixture(scope="package")
+def host_with_ring0_tests(host_with_saved_yum_state: Host):
```

Comment:
What's the rationale of having this fixture in `conftest` ?  That seems really specific to `test_ring0` (which incidentally includes another fixture)

Reply:
Moved fixture to `test_ring0`.